lib/honeybadger.ex

```diff
@@ -161,7 +170,26 @@ defmodule Honeybadger do
       hostname: :inet.gethostname |> elem(1) |> List.to_string,
       origin: "https://api.honeybadger.io",
       project_root: System.cwd,
-      use_logger: true,
-      environment_name: nil]
+      use_logger: true]
+  end
+
+  defp get_and_set_environment_name!(config) do
+    case System.get_env("MIX_ENV") do
```

Comment:
I'm thinking that maybe MIX_ENV should be the default, but if the honeybadger config option is set explicitly, that should be preferred. It looks like this will always prefer the environment variable. @rbishop thoughts?